src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -320,9 +320,42 @@ object SampleWdl {
              Array[File] outs = ["out", "out2"]
           }
         }
+        task C {
+          command {
+            cat > out <<END
+            (_)
+             !_________________________________________
+             !*  *  *  *  * |##########################|
+             ! *  *  *  *  *|                          |
+             !*  *  *  *  * |##########################|
+             ! *  *  *  *  *|                          |
+             !*  *  *  *  * |##########################|
+             ! *  *  *  *  *|                          |
+             !*  *  *  *  * |##########################|
+             !~~~~~~~~~~~~~~~                          |
+             !#########################################|
+             !                                         |
+             !#########################################|
+             !                                         |
+             !###################################JGS###|
+             !~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+             !
```

Comment:
Yeah the artist apologized for leaving out 15 stars for supposedly aesthetic reasons

http://www.chris.com/ascii/joan/www.geocities.com/SoHo/7373/flag.html